docs/keepassxc-browser-manual.md

```diff
@@ -0,0 +1,333 @@
+</div>
+
+At first all input fields are highlighted. Click on the field you want to use as a username field or skip this step if no username field is required.
+Next, choose a password field and in the last step confirm your selection.
+
+<div style="text-align: center;">
+<img alt="KeePassXC-Browser custom credential fields" style="max-width: 100%;"
+src="{{ site.baseurl }}/images/browser-migration/manual_custom_credential_fields.png">
+</div>
+
+Additional fields called *String Fields* can be selected before the confirm. This functionality is explained in [1.7. Fill additional values using String Fields](#17-fill-additional-values-using-string-fields).
+
+Next time you open the page KeePassXC-Browser will use the defined input fields and no longer auto detect them. The custom credential fields can be
+modified or removed from *Settings -> Custom credential fields*, or by reopening the selection from the popup and selecting *Discard selection*.
+
+### 1.5. Create new or modify existing credentials
+
+KeePassXC-Browser is able to detect form submits. If unsaved or modified credentials are detectd, the KeePassXC-Browser icon starts blinking red.
+When clicked, it remains red until you add, update or dismiss the detected changed. It is also possible to put the site to an ignore list when no 
+credential changes are informed in the future.
+
+<div style="text-align: center;">
+<img alt="KeePassXC-Browser remember credentials popup" style="max-width: 100%;"
+src="{{ site.baseurl }}/images/browser-migration/manual_remember_credentials_popup.png">
+</div>
+
+If you have multiple credentials for a page, clicking *Update* opens a list of all available entries on which you can select the outdated entry.
+If the username matches with username(s) of the available credentials, this entry will be marked bold.
+
+<div style="text-align: center;">
+<img alt="KeePassXC-Browser remember multiple credentials" style="max-width: 100%;"
+src="{{ site.baseurl }}/images/browser-migration/manual_remember_multiple_credentials.png">
+</div>
+
+New entries are stored in a separate group *KeePassXC-Browser Passwords* in the database.
+
+In *Settings -> General* the following settings are related to this feature:
+- *Blink Time* (The maximum time icon blinks after detecting a credential change)
+- *Redirect Offset* (The time icon blinks after a page redirect)
+- *Redirect Allowance* (How many redirects are allowed before deactivating the blinking icon) 
+- *Show a notification when new credentials can be saved to the database.*
+- *Save domain only.* (When saving new credentials save only the domain instead of full URL)
+
+Please notice that detecting the form changes might not work with AJAX requests.
+
+### 1.6. Choose credentials for HTTP Basic Auth requests
+
+KeePassXC-Browser can try automatically fill HTTP Basic Auth dialogs. If only one credentials are available, the dialog will be filled automatically.
```

Comment:
"try automatically fill" → "try to automatically fill"
"one credentials are" → "one set of credentials is"?